packages/backend-common/src/logging/rootLogger.ts

```diff
@@ -38,3 +26,35 @@ export function getRootLogger(): winston.Logger {
 export function setRootLogger(newLogger: winston.Logger) {
   rootLogger = newLogger;
 }
+
+export function createRootLogger(
+  options: winston.LoggerOptions = {},
+): winston.Logger {
+  const logger = winston.createLogger({
+    level: process.env.LOG_LEVEL || 'info',
+    format: winston.format.combine(
+      winston.format.errors({
+        stack: process.env.LOG_STACKTRACE !== undefined || false,
```

Comment:
Was considering this should be enabled by default, but didn't want to potentially introduce unwanted behaviour.

Reply:
would it make sense to always have this if `LOG_LEVEL=debug` instead of having a separate var?

Reply:
Yeah, I didn't really want to introduce a new env variable, but I don't think I'd want to set the level to debug just to get stack traces for things normally logged as level `error`. My preference would be to have this on always TBH. If only has an effect when an `Error` object is passed as either the message or an additional arg (meta field). https://github.com/winstonjs/logform/blob/master/errors.js. I can't really see a reason why you wouldn't want the full stack trace to be included when ever an error object is passed along. Thoughts?

Reply:
So the above is a completely moot point. Looks like `stack` is included by default https://github.com/winstonjs/winston/blob/master/lib/winston/logger.js#L244. Will just remove this and pretend I never added it 😏